src/haz3lweb/Update.re

```diff
@@ -528,7 +528,7 @@ let rec apply =
       switch (result) {
       | NoElab
       | Evaluation(_) => ()
-      | Stepper(_stepper) => print_endline("Hello world!")
+      | Stepper(_stepper) => CoqExport.exportCoq(_stepper.previous)
```

Comment:
small style thing: an underscore at the start of a variable usually means that the variable is unused.

Reply:
Thanks! Fixed in the next commit